src/App.tsx

```diff
@@ -15,19 +15,25 @@ import LoginControl from "./LoginControl"
 
 import "./App.scss"
 import logoDark from "./images/logo-dark.png"
+import { connect } from "tls"
 
 interface AppState {
-  loggedIn: boolean
+  loggedIn: string
 }
 
 export default class App extends React.Component<unknown, AppState> {
   constructor(props: unknown) {
     super(props)
     this.state = {
-      loggedIn: localStorage.getItem(consts.brigadeAPITokenKey) ? true : false
+      loggedIn: ""
     }
   }
 
+  identifyUser = async () => {
+    const userId = (await getClient().authn().whoAmI()).id
```

Comment:
Ofcourse ! This will fail and there should be a try catch block and code will automatically send the control to catch because of request failure. but there aren't any try catch blocks in the code yet so i decided not to add one.

I can add one. Kindly tell me what should I put in the catch ? Should I just put a console.log or just leave it empty ? Just to mention eslint wasn't allowing me to put an empty catch block so we have to disable that eslint rule if we want to use an empty catch block over there.

Reply:
Just to be very clear... I don't want the absence of try/catch blocks throughout the code to be mistaken for me not caring about error-handling. It has a lot more to do with me not fully understanding error boundaries in react yet.

So feel free to add a try/catch block here.

Re: empty catch blocks -- I think ESLint might currently not be allowing _any_ empty blocks (catch or otherwise). That rule probably should change.

Last, there's probably another thing you can do here to avoid the error in the first place -- if we don't have a token in local storage, we shouldn't bother making this call.

Reply:
Understood ! I will add a check for token before making the call.